contracts/hyphen/LiquidityProviders.sol

```diff
@@ -120,6 +120,10 @@ contract LiquidityProviders is
         return currentLiquidity[tokenAddress];
     }
 
+    function setCurrentLiquidity(address tokenAddress, uint256 amount) public onlyLiquidityPool {
+        currentLiquidity[tokenAddress] = amount;
```

Comment:
We also need to decrease current liquidity when LP Fee is claimed, during liquidity removal or claimFee